src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -79,3 +79,11 @@ admin_orders_add_payment:
     _controller: PrestaShopBundle:Admin/Sell/Order/Order:addPayment
   requirements:
     orderId: \d+
+
+admin_orders_change_currency:
+  path: /{orderId}/change-currency
```

Comment:
```suggestion
  path: /{orderId}/currency/change
```

Reply:
Why so? :thinking:

Reply:
Because of URI scopes.
`/admin` => You're in admin page
`/admin/order/` => You're in order context
`/admin/order/2` => You select a specific order
`/admin/order/2/currency` => You select the currency for this order
`/admin/order/2/currency/change` => You change the currency for this order.

This is exactly what you do for Advanced parameters
You didn't do `/import-data`, `/import-file-upload` `/import-file-delete`, you split it :)

It's always better to split URIs when you can.
More because you can have ten or twenty things after `/admin/order/2/*` in the future.

Reply:
Perfect. :ok_hand: